projects/core/src/organization/store/reducers/index.ts

```diff
@@ -194,3 +200,17 @@ export const reducerProvider: Provider = {
   provide: reducerToken,
   useFactory: getReducers,
 };
+
+export function clearOrganizationState(
+  reducer: ActionReducer<any>
+): ActionReducer<any> {
+  return function (state, action) {
+    if (action.type === AuthActions.LOGOUT) {
+      state = undefined;
+    }
+
+    return reducer(state, action);
```

Comment:
BUG: to look into, when in an order, and logout, the list and the whole organization state did get reset, but the specific order is still in the state

Reply:
scratch that, I will create a new bug ticket for this, so we do not block the order-approval